### PR TITLE
Fix social media sharing

### PR DESCRIPTION
**Pull request by lantelyes**
#### What does this PR do?
Updates the sharing copy for twitter

#### Description of Task to be completed?
Fix the hashtag copy error when sharing via twitter on donation and pension pages

#### How should this be manually tested?
Go to the various donation/pension pages and click the twitter share button

#### Pull Request Type (check one)
- [ ] Feature
- [x] Bug Fixes
- [ ] Chore

#### Developer Checklist
- [x] New code has been scanned for ESlint issues
- [ ] I have tested my code locally ( See note below )
- [x] I made sure that there are no other pull requests that solve for this issue
- [x] My code follows the same structure within the guidelines

Note:
This is WIP, there seems to be no data for movements and protestors in the donation page, and policies in the pension page. So I have not been able to test those

### CODE REVIEW

**Comment by Easybuoy** [comment]
Tested, and it only works for Twitter, is there any plan for doing for the other social media platforms? @lantelyes

**Comment by lantelyes** [comment]
I can add copy to the other platforms as well, should it be the same as the copy for twitter?

**Comment by Easybuoy** [comment]
> I can add copy to the other platforms as well, should it be the same as the copy for twitter?

Yes, It should be

**Comment by Easybuoy** [comment]
Also noticed the profile detail sharing hasn't been done yet as well

**Comment by Easybuoy** [comment]
Good job, it looked great on Twitter

**Comment by lantelyes** [comment]
Thanks! And thanks for the clarification. I'll be home in an hour or so, and update the other social platforms with this copy, and complete the profile detail sharing

**Comment by lantelyes** [comment]
@Easybuoy I've updated all the links to use the same copy, and added the copy to the profile details page (I used the same copy as for victim donations)

**Comment by Easybuoy** [comment]
Have you been able to test this? https://github.com/Say-Their-Name/say-their-names-web/issues/62#issuecomment-641431288

**Comment by Easybuoy** [comment]
> @Easybuoy I've updated all the links to use the same copy, and added the copy to the profile details page (I used the same copy as for victim donations)

Looks good

**Comment by ashleygraf101** [comment]
Whatsapp doesn't have hashtags, but it could be a great visual point of interest in group chats.

**Comment by lantelyes** [comment]
> Have you been able to test this? [#62 (comment)](https://github.com/Say-Their-Name/say-their-names-web/issues/62#issuecomment-641431288)

Not yet, @devingray I still don't see the data in the database, could you make sure it was published?

File: src/pages/PetitionDetail.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
 
 const { apiBaseUrl } = config;
 
+const getSocialCopy = (petitionDetails) => {
+  let socialCopy;
+  if (petitionDetails.person) {
+    socialCopy = `Help bring justice to ${petitionDetails.person.full_name} by signing this petition and put pressure on the authorities to do the right thing. If we show up in numbers, we can win. 
+#SayTheirNames #BlackLivesMatter #${petitionDetails.person.full_name.replace(/\s+/g, '')}`;
+  } else {
+    socialCopy = 'Sign this petition to demand policy change and bring media attention to the number of people who want action. #BlackLivesMatter #SayTheirNames';
```
Comment (ashleygraf101) [inline src/pages/PetitionDetail.js:22]: Perhaps change to 'bring attention to how many of us want action'?

File: src/pages/DonationDetail.js [hunk 2]
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
 
 const { apiBaseUrl } = config;
 
+const getSocialCopy = (donationDetails) => {
+  let socialCopy;
+  if (donationDetails.person) {
+    socialCopy = `Join me in donating to ${donationDetails.person.full_name}’s fund, and supporting their family’s fight for justice. 
+#SayTheirNames #BlackLivesMatter #${donationDetails.person.full_name.replace(/\s+/g, '')}`;
+  } else {
+    socialCopy = `The movement goes on. Join me in supporting ${donationDetails.fund} and let’s help fuel the campaign for justice together. #BlackLivesMatter #SayTheirNames`;
```
Comment (ashleygraf101) [inline src/pages/DonationDetail.js:22]: Remove 'help'. In retrospect qualifiers weaken the message.

File: src/pages/DonationDetail.js [hunk 3]
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
 
 const { apiBaseUrl } = config;
 
+const getSocialCopy = (donationDetails) => {
+  let socialCopy;
+  if (donationDetails.person) {
+    socialCopy = `Join me in donating to ${donationDetails.person.full_name}’s fund, and supporting their family’s fight for justice.
```
Comment (ashleygraf101) [inline src/pages/DonationDetail.js:19]: Perhaps add 'We remember (full_name}' to the start, and then change from '{full_name}'s fund' to 'their memorial fund'.

**Comment by lantelyes** [comment]
> I just made some copy suggestions. Code looks good!

Good suggestions! made the changes

File: src/pages/DonationDetail.js [hunk 4]
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
 
 const { apiBaseUrl } = config;
 
+const getSocialCopy = (donationDetails) => {
+  let socialCopy;
+  if (donationDetails.person) {
+    socialCopy = `'We remember ${donationDetails.person.full_name}. Join me in donating to thier memorial fund, and supporting their family’s fight for justice.
```
Comment (ashleygraf101) [inline src/pages/DonationDetail.js:19]:
```suggestion
    socialCopy = `'We remember ${donationDetails.person.full_name}. Join me in donating to their memorial fund, and supporting their family’s fight for justice. 
```

**Comment by Easybuoy** [comment]
I asked Devin, he said he's pushed the changes, but if there're still issues he'd fix tomorrow. You can reach out to him on slack if they're still issues

**Comment by Easybuoy** [comment]
I think we should merge this, we'd fix any other issue that comes up in another pull request.

This is MVP critical

**Pull request merged**